### DISCUSSION TITLE
Readability in menus

### DISCUSSION THREAD

**Discussion by Irodzuita** [gotson/komga#1737] [Ideas]
**Suggestions**:

- Based on the ComicInfo formatting from the Anasi-project I'd like to have Characters and Locations included in the indexing so that they do not have to be included in the tag list and making it hard to navigate. This would also be helpful for downloaders to sort information differently - e.g. if you have a library of parodies then location could be used as the parody tag to, again, decrease clutter in tags. Currently around 85-90% of my tag list is parodies and characters so it would help me immensely to have a place to sort them so I can actually get to the tags. 

- For listing tags inside of the book information menu I'd like to see them be shown in a multi-line format so that they are easier to read without scrolling through the list. This one could just be me but I'd find it easier to quickly get the gist of a new book the my downloader grabbed if it was formatted this way.

- In read list tab when selecting all (or multiple) I'd like to have a remove from reading list button instead of only the delete. This could be in place of "add to reading list" possibly as personally I'd find it much more likely someone would be bulk removing than switching reading lists but any method of adding it works for me.

Thank you for reading my ideas

**Comment by gotson** [comment]
First 2 point are already covered in #1551 #1530 

Not sure to understand the 3rd point

**Reply by Irodzuita** [reply]
#1551 partially covers the first point though seems to lack only be characters and not locations too. #1530 however doesn’t talk about the same thing. That is asking about a different tag search menu that is easier to read while I am asking for the tags listed inside of the book information screen to be changed. The book info screen being the one with pages, tags, author, file etc. I’d like to see on that screen the tags be viewed in multiple lines instead of having to scroll to the right to see all of them. 

For the last point I was talking about the reading list screen on the top banner with delete, mark as read/unread, etc. when multiple books are selected and asking for there to be another option to “remove from reading list” so we can remove multiple books at once from the list

**Reply by gotson** [reply]
i meant to quote https://github.com/gotson/komga/discussions/1585 and not #1530



> For the last point I was talking about the reading list screen on the top banner with delete, mark as read/unread, etc. when multiple books are selected and asking for there to be another option to “remove from reading list” so we can remove multiple books at once from the list

OK i understand, but that's not how read lists are updated. You need to get into edit mode to remove books.

**Comment by gotson** [comment]
Tracking entries containing multiple different unrelated ideas is not easy, please open separate discussions.
